ld_script_test.ld

```diff
@@ -52,6 +52,7 @@ SECTIONS {
 
         src/m4a.o(.bss.code);
 
+        src/*.o(common_data);
         src/*.o(COMMON);
         gflib/*.o(COMMON);
```

Comment:
This line now needs to be removed as well, doesn't it?

Reply:
It couldn't build without it.
![image](https://github.com/user-attachments/assets/0c0d2687-7e02-4650-b95e-718ea0aeb4c4)

Reply:
GitHub is unclear, I attached the comment to the gflib line, which is deleted in the other ld scripts

Reply:
oh, `gflib/*.o(COMMON);`. got it 👍

Reply:
Not sure why you marked this as resolved because the line is still there :)

Reply:
Nevermind, GitHub was playing tricks on my and showed a cached version, sorry